internal/webpagetest.py

```diff
@@ -749,6 +749,9 @@ def notify_test_started(self, job):
         """Tell the server that we have started the test. Used when the queueing isn't handled directly by the server responsible for a test"""
         if 'work_server' in job and 'Test ID' in job:
             try:
+                if 'saas_test_id' in job:
+                    logging.info("Test id: {0} started".format(job['saas_test_id']))
```

Comment:
I'd like to find a better way to format the saas_test_id in, but haven't yet.

Reply:
so, I have

`globals()['test_id'] = job['Test ID'] if job['saas_test_id'] is None else job['saas_test_id']` going on, which allows for the the logging of the saas_test_id normally and then fallback to the Test ID otherwise. That _feels_ like it should handle this.

That being said, I do not believe there is a regular `Test ID`